pep-0680.rst

```diff
@@ -2,13 +2,13 @@ PEP: 680
 Title: tomllib: Support for parsing TOML in the Standard Library
```

Comment:
```suggestion
Title: tomllib: Support for Parsing TOML in the Standard Library
```

or

```suggestion
Title: tomllib: Support for parsing TOML in the standard library
```

I suggest either use Title Case or Sentence case consistently (title case seems to predominate for PEP titles, though both can be found). To note, per the [standard library docs](https://docs.python.org/3/library/), it isn't considered a proper noun as it is not capitalized there, if that was where the confusion came from.